packages/mdc-textfield/foundation.js

```diff
@@ -43,6 +45,11 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       deregisterInputInputHandler: (/* handler: EventListener */) => {},
       registerInputKeydownHandler: (/* handler: EventListener */) => {},
       deregisterInputKeydownHandler: (/* handler: EventListener */) => {},
+      registerInputPointerDownHandler: (/* evtType: string, handler: EventListener */) => {},
```

Comment:
Why is this specifically named `registerInputPointerDownHandler` if it accepts any event type?  If we're going to do this, we might as well remove all of the specific `registerInputFooHandler` methods and just have `registerInputHandler(evtType, handler)`.  That would be an even bigger adapter change (but maybe a welcome one to reduce 6 methods to 2).

Also, your documentation in the README is inconsistent with this; the README doesn't mention it takes `evtType`.

Reply:
In practice all it does take are pointer events—the name is there to reinforce that. I think we want to keep one method per registration. A user may need to override some subset of these and not all of them.

I agree with the documentation oversight and fixed it.